### PR TITLE
fixed shared star rating

### PR DESCRIPTION
**Pull request by campbeb3**
had to make changes to get the star rating to function

### CODE REVIEW

File: bike_kollective/lib/bike_details_screen.dart [hunk 1]
```diff
@@ -27,7 +29,9 @@ class BikeDetailsScreen extends ConsumerWidget {
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
             BikeDetailsView(
-                bike: bike), // Include the BikeDetailsView component
+              bike: bike, // Include the BikeDetailsView component
```
Comment (kenneth-tucker) [inline bike_kollective/lib/bike_details_screen.dart:32]: Could we just pass in the bike and get bike.rating() inside of BikeDetailsView? No need for this extra arg

File: bike_kollective/lib/bike_details_view.dart [hunk 2]
```diff
@@ -4,8 +4,9 @@ import 'package:flutter_rating_bar/flutter_rating_bar.dart';
 
 class BikeDetailsView extends StatelessWidget {
   final BikeModel bike;
+  final double? rating; // Add a rating parameter
```
Comment (kenneth-tucker) [inline bike_kollective/lib/bike_details_view.dart:7]: Remove this. You can get the rating from bike, no need for extra member/constructor param

File: bike_kollective/lib/data/model/bk_document_reference.dart [hunk 3]
```diff
@@ -43,4 +43,20 @@ class BKDocumentReference {
   bool isFake() {
     return (fakeDocumentId != null);
   }
+ Future<Map<String, dynamic>?> getData() async {
```
Comment (kenneth-tucker) [inline bike_kollective/lib/data/model/bk_document_reference.dart]: Please remove this, we don't need it

Comment (campbeb3) [reply bike_kollective/lib/data/model/bk_document_reference.dart]: Removed, thanks!

File: bike_kollective/lib/bike_details_screen.dart (hunk 1 above)
Comment (campbeb3) [reply bike_kollective/lib/bike_details_screen.dart:32]: Made change, but noticed all of my bikes now had same 2-star rating.

**Review by kenneth-tucker — APPROVED** [review]

**Review by anisar17 — APPROVED** [review]

**Comment by campbeb3** [comment]
Closed PR after commenting out my stuff, but Star Rating no longer works

**Pull request merged**